Godeps/Godeps.json

```diff
@@ -370,6 +370,11 @@
 			"Rev": "63ce630574a5ec05ecd8e8de5cea16332a5a684d"
 		},
 		{
+			"ImportPath": "github.com/aws/aws-sdk-go/service/kms",
```

Comment:
This should be in the same commit that has the code.  Shaped like `bump(github.com/aws/aws-sdk-go): 63ce630574a5ec05ecd8e8de5cea16332a5a684d`.  Also, thanks for picking the same level as everything else.

Reply:
Fixed, thanks